hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/airflow/AirflowDagFacade.java

```diff
@@ -16,33 +16,68 @@
 
 package io.hops.hopsworks.common.dao.airflow;
 
+import javax.annotation.Resource;
 import javax.ejb.Stateless;
 import javax.ejb.TransactionAttribute;
 import javax.ejb.TransactionAttributeType;
-import javax.persistence.EntityManager;
-import javax.persistence.PersistenceContext;
-import javax.persistence.TypedQuery;
+import javax.sql.DataSource;
 import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This is special case of a facade. It does not use the EntityManager.
+ *
+ * DO NOT invoke the DataSource directly!
+ *
+ * Look on other facades for examples on how to access the database.
+ */
 
 @Stateless
-@TransactionAttribute(TransactionAttributeType.REQUIRED)
+@TransactionAttribute(TransactionAttributeType.NOT_SUPPORTED)
```

Comment:
@kouzant I know why we do that but we can add a comment explaining why this facade is different than others to assist people using this code in the future

Reply:
Done. I added some more detailed information.